lib/venue.rb

```diff
@@ -1,2 +1,30 @@
 class Venue
+  attr_reader :name, :capacity
+  attr_accessor :patrons
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+
+  end
+  def add_patron(patron)
+    @patrons << patron
+  end
+  def yell_at_patrons
+    @patrons.map{|patron| patron.upcase}
+  end
+  def over_capacity?
+    return true if @patrons.length > @capacity
```

Comment:
You can get rid of everything before `@patrons.length > @capacity` as well as the `false`. Because you are using the `>` operator it will return a boolean

Reply:
Oh okay thats cool. I did not know that.